client/state/themes/selectors.js

```diff
@@ -603,3 +603,26 @@ export function isThemePurchased( state, themeId, siteId ) {
 	const sitePurchases = getSitePurchases( state, siteId );
 	return some( sitePurchases, { productSlug: 'premium_theme', meta: themeId } );
 }
+
+/**
+ * Returns themeId to preview for site.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Object}         Last query
+ */
+export function getThemeForPreviewData( state ) {
+	const themeData = get( state.themes.previewingTheme, 'themeData', null );
+	const themeOptions = get( state.themes.previewingThemeOptions, 'themeOptions', null );
+	return { themeData, themeOptions };
```

Comment:
Can we have separate selectors for these, instead of a shared one? They're getting a bit too [nested](https://github.com/Automattic/wp-calypso/pull/11167/files#diff-8973cec3a24a1ea3d7e46f0a0543e7a8R55) 😄